pandas-stubs/core/frame.pyi

```diff
@@ -1628,24 +1643,7 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
     def isin(self, values: Iterable | Series | DataFrame | dict) -> Self: ...
     @property
     def plot(self) -> PlotAccessor: ...
-    def hist(
```

Comment:
Code has changed and now points directly to https://github.com/pandas-dev/pandas/blob/c888af6d0bb674932007623c0867e1fbd4bdc2c6/pandas/plotting/_core.py#L145-L268